corehq/apps/app_manager/static/app_manager/js/app_diff.js

```diff
@@ -47,13 +47,13 @@ hqDefine('app_manager/js/app_diff.js', function () {
     var reverse = hqImport('hqwebapp/js/urllib.js').reverse;
     var sanitize = DOMPurify.sanitize;
 
-    var init = function(selector, appIdOne, appIdTwo, options) {
+    var init = function(selector, options) {
         var $el = $(selector);
 
         if (!$el.length) {
-            throw new Error(selector + ' does not resolve to an element');
+            window.console.warn(selector + ' does not resolve to an element');
```

Comment:
it's pretty bad to break the release page if the element for this doesn't exist, so decided to change it to a warning